packages/k8s/src/hooks/prepare-job.ts

```diff
@@ -166,14 +167,20 @@ function createPodSpec(
   const podContainer = {
     name,
     image: container.image,
-    command: [container.entryPoint],
-    args: container.entryPointArgs,
     ports: containerPorts(container)
```

Comment:
What you guys think about adding same sort of check for ports like 
`if (container.portMappings) podContainer.ports = containerPorts(container)`
By default, I don't need to expose ports for containers in same pod, and currently if I don't specify a port it fails.

Reply:
Hey @claveaur,

Can you please submit an issue with the repro? It should be part of a different PR. :relaxed: